redirects.yaml

```
@@ -11,7 +11,8 @@ careers/tech-ops/?: "/careers/support-engineering"
 careers/techops/?: "/careers/support-engineering"
 careers/design/?: "/careers/web-and-design"
 careers/departments/?: "/careers"
-careers/commercial-ops/?: "/careers/operations"
+careers/commercial-ops/?: "/careers/commercial-operations"
+careers/operations/?: "careers/commercial-operations"
```

```suggestion
careers/operations/?: "/careers/commercial-operations"
```
This redirect currently goes to `/careers/careers/commercial-operations` which 404s. This might be due to the missing /